tests/foreman/cli/test_discovery.py

```diff
@@ -1,17 +1,20 @@
 # -*- encoding: utf-8 -*-
 """Test class for CLI Foreman Discovery"""
-from robottelo.decorators import stubbed
+from robottelo.decorators import run_only_on, stubbed, tier3
 from robottelo.test import CLITestCase
 
 
 class DiscoveryTestCase(CLITestCase):
     """Implements Foreman discovery CLI tests."""
 
+    @run_only_on('sat')
     @stubbed()
-    def test_positive_discovered_host_facts(self):
-        """@Test: Check all facts of discovered hosts are correctly displayed
+    @tier3
+    def test_custom_facts_pxeless_discovery(self):
```

Comment:
As per issue #3083 , we need to specify whether this is positive or negative testcase.
Hence the second word needs to be `test_positive` or `test_negative`.